submissions/migrations/0003_submission_status.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='status',
+            field=models.CharField(default=None, max_length=1, null=True, choices=[(b'D', b'Deleted'), (b'A', b'Active')]),
```

Comment:
As discussed, since we are warning about migration time anyway, we'll go for Active as default.